extensions/samples/electron-webpack/CHANGELOG.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "electron-msal-node-extensions",
```

Comment:
This doesn't need a changelog, should be removed from lerna

Reply:
Just saw these comments. All should be addressed in #4112